HHS/OPRE-OPS
# Fixing small visual error.

## What changed

PRocurement Shop Select had minor error in display after the change to fix all the math. This corrects the error.

## Issue

N/A

## How to test

Open the UI, go to where you can select the procurement shop (like on an agreement). Verify that the percentage is correct (in the small footnote under the selection box).

## Screenshots

<img width="439" alt="Screenshot 2023-10-02 at 15 06 18" src="https://github.com/HHS/OPRE-OPS/assets/881707/e7c037ab-6b10-4107-a070-5b1c5b8dd8aa">

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
